### DISCUSSION TITLE
Can tailwind in leptos can compile this code or not ?

### DISCUSSION THREAD

**Discussion by ALEZ-DEV** [leptos-rs/leptos#2430] [Q&A] [answered]
Can tailwind in the leptos compiler can build this code, because de class is incomplete ?
`<div class={format!("h-14 col-start-{}", i)}></div>`

**Comment by dglsparsons** [comment, marked as answer]
Tailwind in JS recommends against doing this, as the classnames are incomplete. 
[source](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 

I don't think there is any reason it would be different in Rust. Realistically, you can't know what classnames are needed ahead of runtime with code written in that manner.

**Reply by ALEZ-DEV** [reply]
okay that's what I was thinking, thanks for the response!
